sdcm/cluster.py

```diff
@@ -2057,6 +2057,23 @@ def scylla_setup(self, disks):
         self.remoter.run('sudo sync')
         self.log.info('io.conf right after setup')
         self.remoter.run('sudo cat /etc/scylla.d/io.conf')
+        self.remoter.run('sudo cat /etc/scylla.d/io_properties.yaml')
+
+        cluster_backend = self.parent_cluster.params.get('cluster_backend')
+        nr_disks = self.parent_cluster.params.get('gce_n_local_ssd_disk_db')
+
+        if cluster_backend == 'gce' and nr_disks >= 4 & nr_disks <= 8:
+            with self.remote_scylla_yaml('/etc/scylla.d/io_properties.yaml') as io_properties:
+                mbs = 1024 * 1024
+                io_properties['disks'][0]['read_iops'] = 680000
+                io_properties['disks'][0]['read_bandwidth'] = 2650 * mbs
+                io_properties['disks'][0]['write_iops'] = 360000
+                io_properties['disks'][0]['write_bandwidth'] = 1400 * mbs
+
+            cpu_set = self.remoter.run(
+                f"PATH=$PATH:/opt/scylladb/bin /opt/scylladb/scripts/perftune.py --tune net --get-cpu-mask --mode sq_split --nic {devname} | /opt/scylladb/scripts/hex2list.py")
+            cpu_set = cpu_set.stdout.strip()
+            self.remoter.sudo(f"""sed 's/"--cpuset.*"/"--cpuset {cpu_set}"/' -i /etc/scylla.d/cpuset.conf""")
```

Comment:
I don't think we should do it, it's a workaround that we will forget one day.

Reply:
I agree,

But without I couldn't test any formal version only master

Do you think we should check scylla version, before doing that ?

Reply:
we decide to remove this